cell_type_prediction/galaxy_cli_use_case/galaxy_credentials.yaml

```diff
@@ -0,0 +1,5 @@
+__default: ebi_cluster
+
+ebi_cluster:
+    key: "d78add8d5ea953a6f925b2def73f1010"
```

Comment:
please delete this key and re-generate it, and avoid putting the galaxy_credentials file that you use under version control. Just add a sample with dummy data.

Reply:
unless that this is already... also, avoid showing the hostname that we use internally.